GoogleForCreators/web-stories-wp
# Add Cross-Origin-Opener header to editor screen

## Context



## Summary

Add Cross-Origin-Opener header to editor screen. Also add cross origin attribute to img, script, iframe and link tags.

## Relevant Technical Choices

Filter avatar, script and style tags before the output buffer, just a little cleaner and make likely to get results. 

## To-do



## User-facing changes



## Testing Instructions



### QA

This PR can be tested by following these steps:

1. In firefox, open in the editor. ( ensure video transcoding experiment is enabled. )
2. Ensure the font loads and avatar is visual in admin bar. 
3. Drag a .mov file into the editor. 
4. See video transcode. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6776
